src/analyze/analyze.c

```diff
@@ -82,6 +86,9 @@ static const char *arg_root = NULL;
 STATIC_DESTRUCTOR_REGISTER(arg_dot_from_patterns, strv_freep);
 STATIC_DESTRUCTOR_REGISTER(arg_dot_to_patterns, strv_freep);
 
+static char *dependency_color[_UNIT_DEPENDENCY_MAX] = {};
+static char *dependency_style[_UNIT_DEPENDENCY_MAX] = {};
```

Comment:
even though these aren't directly cmdline params (why not btw, might make sense to expose them that way too. i.e. `--dependenc-color=After:red` or so), I think they are very similar to those, and hence should be prefixed with `arg_`, which is how we generally prefix our process-wide runtime parameters.

Reply:
Added option --dependency-color-style and prefixed with arg_